src/components/DocumentationTopic.vue

```diff
@@ -32,8 +32,10 @@
           :objcPath="objcPath"
           :swiftPath="swiftPath"
         />
-        <LinkableHeading v-if="enableMinimized" class="minimized-summary">Summary</LinkableHeading>
-        <Title v-else :eyebrow="roleHeading">
+        <Title
+          :eyebrow="enableMinimized ? '' : roleHeading"
```

Comment:
This works since `!!''` evaluates to `false`, but I think using `null` or `undefined` instead of the empty string would make it a little more clear that we're purposefully omitting the eyebrow value in this case.